src/mscorlib/shared/System/ReadOnlySpan.Fast.cs

```diff
@@ -266,5 +279,7 @@ public T[] ToArray()
             Buffer.Memmove(ref Unsafe.As<byte, T>(ref destination.GetRawSzArrayData()), ref _pointer.Value, (nuint)_length);
             return destination;
         }
+
+        private static T s_nullPtr = default;
```

Comment:
This should not be static.

Reply:
I wonder if it is worth caching at all - vs. computing.

Reply:
It is caching the wrong thing also.